pages/index.tsx

```diff
@@ -319,15 +326,16 @@ const AllPlugins: React.FC = () => (
 )
 
 const Home: NextPage = () => (
-  <Box sx={{ flexGrow: 1 }}>
-    
-    <TopBar />
-    <Blurb />
-    <br />
-    <MostPopularPlugins />
-    <br />
-    <AllPlugins />
-  </Box>
+    <Box sx={{ flexGrow: 1 }}>
+      <TopBar />
+      <Container maxWidth="xl">
```

Comment:
Having cards up against the edges of the screen felt a little odd on large screens, so contained it to a max of xl width. Might be worth adding padding to the MostPopularPlugins/AllPlugins components too, idk.